Tinylog/View Controllers/Lists/AddListViewController.swift

```diff
@@ -40,40 +41,52 @@
         super.viewDidLoad()
 
         setupNavigationBarProperties()
-
+        setupUITableView()
+        setupNavigationItem()
+        setupMenuColorsView()
+        setupViewMode()
+    }
+    
+    private func setupUITableView() {
+        tableView.isScrollEnabled = false
         tableView.backgroundColor = UIColor(named: "mainColor")
         tableView.separatorColor = UIColor(named: "tableViewSeparator")
-
+        tableView.register(TextFieldCell.self,
+                           forCellReuseIdentifier: TextFieldCell.cellIdentifier)
+    }
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/03eace86b303bd47c13e99b11bd2b1214da5c26b/Tinylog/View Controllers/Lists/AddListViewController.swift#L57"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AddListViewController.swift:57</code></td>
    </tr>
  </tbody>
</table>